pyproject.toml

```diff
@@ -70,6 +70,7 @@ optional-dependencies.tests = [
   "pytest",
   "pytest-cov",
   "pytest-timeout",
+  "pytest-xdist",
```

Comment:
If the Makefile is going to install pytest-xdist by itself, and we're not using it anywhere else, I don't think this needs to be in pyproject.toml?

Reply:
Depends on if we decide to make it the default or not, really.